project.clj

```diff
@@ -1,6 +1,10 @@
 (defproject fundingcircle/jackdaw "_"
   :description "A Clojure library for the Apache Kafka distributed streaming platform."
 
+  :scm {:name "git" :url "https://github.com/fundingcircle/jackdaw"}
```

Comment:
What is this for?

Reply:
From https://github.com/cljdoc/cljdoc/blob/master/doc/userguide/faq.md#how-do-i-set-scm-info-for-my-project